doc/2-interface/asset-list.md

```diff
@@ -3,10 +3,20 @@
 ![instruments window](instruments.png)
 
 Buttons from left to right:
-- **Add**: Creates a new, default instrument.
+- **Add**: pops up a menu to select which type of instrument to add.
+  - If the "Display instrument type menu when adding instrument" setting is disabled, this skips the menu and creates an instrument according to the chip under the cursor.
```

Comment:
The menu only appears if there are multiple viable instrument types. For example, it will appear on Genesis, but not on Game Boy